larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -342,81 +300,52 @@ float GetTensorValue(const TfLiteTensor* tensor, int index) {
           tensor->params.zero_point);
 }
 
-void SetupQuantization(TfLiteContext* context, TfLiteNode* node,
-                       TfLiteBConv2DParams* params) {
-  const auto* output = GetOutput(context, node, 0);
-  const auto* post_activation_multiplier = GetInput(context, node, 2);
-  const auto* post_activation_bias = GetInput(context, node, 3);
-
-  // Not required when writing bitpacked output
-  if (output->type == kTfLiteInt32) return;
-
-  // Step 1
-  // If the post_activation_ data is stored in int8, then convert it to float
-  if (post_activation_multiplier->type != kTfLiteFloat32 ||
-      post_activation_bias->type != kTfLiteFloat32) {
-    for (int i = 0; i < params->channels_out; ++i) {
-      params->scaled_post_activation_multiplier[i] =
-          GetTensorValue(post_activation_multiplier, i);
-      params->scaled_post_activation_bias[i] =
-          GetTensorValue(post_activation_bias, i);
-    }
-  }
```

Comment:
Another code path that's no longer relevant; with float32 output we don't allow non-float mul/bias (there's a tensor type check in Prepare).

Reply:
The body of `SetupQuantization` has been moved into `OneTimeSetup`.